examples/README.md

```diff
@@ -15,3 +15,38 @@ $ kubectl create -f pod.yaml
 Other helper scripts:
 * `logs.sh` output of the plugin
 * `exec-bash.sh` logs into the plugin's container and runs bash
+
+## How to test RBD Snapshot feature
+
+Before continuing, make sure you enabled the required [feature gate](https://kubernetes-csi.github.io/docs/Setup.html#csi-volume-snapshot-support) in your Kubernetes cluster.
+
+In the `examples/rbd` directory you will find two files related to snapshots: [snapshotclass.yaml](./rbd/snapshotclass.yaml) and [snapshot.yaml](./rbd/snapshot.yaml).
+
+Once you created your RBD volume, you'll need to customize at least `snapshotclass.yaml` and make sure the `monitors` and `pool` parameters match your Ceph cluster setup. If you followed the documentation to create the rbdplugin, you shouldn't have to edit any other file.
+
+After configuring everything you needed, deploy the snapshot class:
+
+```bash
+kubectl create -f snapshotclass.yaml
+kubectl create -f snapshot.yaml
```

Comment:
we need some explanation for `kubectl create -f snapshot.yaml`, this creates the snapshot from existing PVC.


we can do like below

```
kubectl create -f snapshotclass.yaml
```
verify snashot class
```
kubectl get volumesnapshotclass
NAME                      AGE
csi-rbdplugin-snapclass   4s
```

* create a snapshot from the existing PVC 
```
kubectl create -f snapshot.yaml
```
* verify snapshot state
```
kubectl get volumesnapshot
NAME               AGE
rbd-pvc-snapshot   6s
```
even we can discribe and show that the snapshot is in ready state

To be sure everything is OK you can run `rbd snap ls [your-pvc-name]` inside one of your Ceph pod.

Reply:
Done